lib/Player.tsx

```diff
@@ -345,6 +333,15 @@ export const Player = forwardRef<PlayerNativeElement, PlayerProps>(
                 toggleStats={toggleStatsOverlay}
               />
             </Layer>
+            {showStatsOverlay && videoProperties !== undefined ? (
+              <Stats
+                api={api}
+                parameters={parameters}
+                videoProperties={videoProperties}
+                open={showStatsOverlay}
```

Comment:
In my opinion, you should _either_ give the user the ability to toggle it from none to big OR from icon to big. Not on => icon => big.

Reply:
IE, remove it from the Settings menu.

Reply:
It's by design ; )